tests/test_core.py

```diff
@@ -8835,6 +8835,14 @@ def test_em_async_js(self):
     self.maybe_closure()
     self.do_core_test('test_em_async_js.c')
 
+  @require_v8
+  def test_externref(self):
+    self.run_process([EMCC, '-c', test_file('core/test_externref.s'), '-o', 'asm.o'])
+    self.emcc_args += ['--js-library', test_file('core/test_externref.js')]
+    self.emcc_args += ['-mreference-types']
+    self.node_args += ['--experimental-wasm-anyref']
```

Comment:
```suggestion
    self.v8_args += ['--experimental-wasm-anyref']
```
as the test runs in v8 and not node

Reply:
Ah thanks.. turns out no argument is needed after all since recent d8 has this enabled by default.